.github/workflows/python-tests.yaml

```diff
@@ -150,6 +150,9 @@ jobs:
           pip install ${{ ! matrix.lower-bound-requirements && '--upgrade --upgrade-strategy eager' || ''}} -e .[dev]
 
       - name: Run tests
+        # Turns off SQLAlchemy 2.0 deprecation warnings. We can remove this once we upgrade to 2.0 
+        env: 
+          SQLALCHEMY_SILENCE_UBER_WARNING: 1
```

Comment:
@desertaxle These should go in the `setup.cfg` pytest configuration instead so they work locally